akka-http-core/src/test/scala/akka/http/impl/engine/client/ConnectionPoolSpec.scala

```diff
@@ -388,7 +388,7 @@ class ConnectionPoolSpec extends AkkaSpec("""
               case x ⇒ fail(x.toString)
             }.toMat(Sink.fold(0)(_ + _))(Keep.both).run()
 
-        Await.result(idSum, 30.seconds) shouldEqual N * (N + 1) / 2
+        Await.result(idSum, 35.seconds) shouldEqual N * (N + 1) / 2
```

Comment:
Hmm, we should try to speed up this test, instead. Any idea why it takes so long?